Hunting Queries/OfficeActivity/sharepoint_downloads.yaml

```
@@ -0,0 +1,44 @@
+// Name: New user agents associated with a clientIP for sharepoint file uploads/downloads
+//
+// Id: e8ae1375-4640-430c-ae8e-2514d09c71eb
+//
+// Description: New user agents associated with a clientIP for sharepoint file uploads/downloads. 
+//
+// DataSource: #OfficeActivity
+//
+// Tactics: #Exfiltration
+//
+
+id: e8ae1375-4640-430c-ae8e-2514d09c71eb
+name: SharePointFileOperation via clientIP with previously unseen user agents
+description: |
+  'New user agents associated with a clientIP for sharepoint file uploads/downloads.'
+requiredDataConnectors:
+  - connectorId: Office365
+    dataTypes:
+      - OfficeActivity
+tactics:
+  - Exfiltration
+relevantTechniques:
+  - T1030
+query: |
+
+  let starttime = 14d;
+  let endtime = 1d;
+  let historicalUA=
+  OfficeActivity
+  | where  RecordType == "SharePointFileOperation"
+  | where Operation in ("FileDownloaded", "FileUploaded")
+  | where TimeGenerated between(ago(starttime)..ago(endtime))
+  | summarize by ClientIP, UserAgent;
+  let recentUA = OfficeActivity
+  | where  RecordType == "SharePointFileOperation"
+  | where Operation in ("FileDownloaded", "FileUploaded")
+  | where TimeGenerated > ago(endtime) 
+  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated) by ClientIP, UserAgent;
```

Suggest maintaining User Id for entity creation
```suggestion
  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated) by ClientIP, UserAgent, UserId;
```

---

As these have been tested on different data sets, I don't want to change core functionality, this could impact so we are leaving out for now.